x-pack/plugins/maps/public/routing/state_syncing/app_state_manager.ts

```diff
@@ -37,7 +46,7 @@ export class AppStateManager {
   getAppState() {
     return {
       query: this._query,
-      savedQuery: this._savedQuery,
+      savedQuery: this._savedQueryId,
```

Comment:
Mind making this `savedQueryId: this._savedQueryId,` to be consistent with updated signatures?

Reply:
Fixed this.